index.html

```diff
@@ -0,0 +1,475 @@
+  - Routers **may**—and typically will—omit CSDL directives and types from the schema they present to clients via introspection
+
+<a name=def-endpoint></a>
+**Endpoints** are running servers which can resolve GraphQL queries against a schema. In this version of the spec, endpoints must be URLs, typically http/https URLs.
+
+<a name=def-subgraph></a>
+**Subgraphs** are the GraphQL schemas which were composed to form the CSDL. A subgraph has:
+  - a name, which must be unique within the CSDL
+  - an [endpoint](#endpoint)
+
+This spec does not place any requirements on subgraph schemas. Generally, they may be of any shape. In particular, subgraph schemas do not need to provide a CSL; neither is it an error for them to do so. Composers **may** place additional requirements on subgraph schemas to aid in composition; composers **should** document any such requirements.
+
+## Structure
+
+A CSDL document must be a valid [GraphQL schema definition language](https://spec.graphql.org/draft/#sec-Type-System) document.
+
+CSDL removes certain SDL features, as described in the [divergence](#divergence-from-graphql-sdl) section. Nevertheless, CSDL consumers **should** be prepared to parse any valid GraphQL SDL. A CSDL document which contains disallowed SDL constructs **should** trigger validation errors after parsing.
+
+CSDL introduces a suite of types and directives. All CSDL-specific types and directives are prefixed with `csdl_` as recommended in [the GraphQL specification](https://spec.graphql.org/draft/#note-ca863).
+
+### Using this spec
+
+CSDL documents **must** contain a `schema` declaration with a [`@using` directive](/using/draft/pre-0).
+
+```graphql
+#::[`@using` directive declaring that the document conforms to csdl v1]
+schema @using(spec: "https://specs.apollo.dev/csdl/v1")
+```
+
+### Forbidden GraphQL IDL features
+
+CSDL documents cannot contain certain [GraphQL type system](https://spec.graphql.org/draft/#sec-Type-System) constructs. Specifically, **extensions are forbidden.** You cannot `extend type` or `extend interface` within a CSDL document. Producers **must** produce documents without extensions.
+
+## Data Model
+TK, draws heavily from federation, a little more general.
+
+### Realized types
+
+A type is *realized* wherever a resolver from a particular *subgraph* returns that type. We use the notation `subgraph::Type` to represent `Type` realized by `subgraph` and `subgraph::Type.field` to represent the resolver for `Type.field` within `subgraph`.
+
+```graphql
+#::[Query with resolvers and realized types]
+query {          # resolver                    | realizes
+                 # ----------------------------|--------------
+  rockets {      # rockets::Query.rockets     -> rockets::Rocket
+    captain {    # rockets::Rocket.astronaut  -> rockets::Astronaut
```

Comment:
I think this line should say `rockets::Rocket.captain -> astronauts::Astronaut`? Or maybe the second half is right but the first isn't.